bigtop-packages/src/common/ambari/patch9-z_bonus.diff

```diff
@@ -0,0 +1,446 @@
+index 1c499b1a..dd682fd2 100644
+--- a/ambari-metrics/pom.xml
++++ b/ambari-metrics/pom.xml
+@@ -28,7 +28,6 @@
+     <module>ambari-metrics-flume-sink</module>
+     <module>ambari-metrics-kafka-sink</module>
+     <module>ambari-metrics-storm-sink</module>
+-    <module>ambari-metrics-storm-sink-legacy</module>
+     <module>ambari-metrics-timelineservice</module>
+     <module>ambari-metrics-host-monitoring</module>
+     <module>ambari-metrics-grafana</module>
+@@ -71,18 +70,6 @@
+       <url>${distMgmtSnapshotsUrl}</url>
+     </snapshotRepository>
+   </distributionManagement>
+-  <repositories>
+-    <repository>
+-      <id>apache-hadoop</id>
+-      <name>hdp</name>
+-      <url>https://repo.hortonworks.com/content/groups/public/</url>
+-    </repository>
+-    <repository>
+-      <id>apache-snapshots</id>
+-      <name>snapshots</name>
+-      <url>https://repository.apache.org/content/repositories/snapshots</url>
+-    </repository>
+-  </repositories>
+   <dependencyManagement>
+     <dependencies>
+       <dependency>
+@@ -323,7 +310,7 @@
+     <dependency>
+       <groupId>org.apache.ambari</groupId>
+       <artifactId>ambari-utility</artifactId>
+-      <version>1.0.0.0-SNAPSHOT</version>
++      <version>2.7.5.0.0</version>
+       <scope>test</scope>
+     </dependency>
+   </dependencies>
+@@ -338,7 +325,7 @@
+         <dependency>
+           <groupId>org.apache.ambari</groupId>
+           <artifactId>ambari-utility</artifactId>
+-          <version>1.0.0.0-SNAPSHOT</version>
++          <version>2.7.5.0.0</version>
+         </dependency>
+       </dependencies>
+     </profile>
+diff --git a/ambari-project/pom.xml b/ambari-project/pom.xml
+index e2010e37..62efdea5 100644
+--- a/ambari-project/pom.xml
++++ b/ambari-project/pom.xml
+@@ -705,7 +705,7 @@
+             <dependency>
+               <groupId>org.apache.ambari</groupId>
+               <artifactId>ambari-utility</artifactId>
+-              <version>1.0.0.0-SNAPSHOT</version>
++              <version>2.7.5.0.0</version>
+             </dependency>
+           </dependencies>
+         </plugin>
+diff --git a/ambari-server/pom.xml b/ambari-server/pom.xml
+index 899c2641..d159e2ec 100644
+--- a/ambari-server/pom.xml
++++ b/ambari-server/pom.xml
+@@ -1776,7 +1776,7 @@
+     <dependency>
+       <groupId>org.apache.ambari</groupId>
+       <artifactId>ambari-serviceadvisor</artifactId>
+-      <version>1.0.0.0-SNAPSHOT</version>
++      <version>2.7.5.0.0</version>
+       <exclusions>
+         <exclusion>
+           <groupId>commons-httpclient</groupId>
+@@ -1789,7 +1789,7 @@
+     <dependency>
+       <groupId>org.apache.ambari</groupId>
+       <artifactId>ambari-utility</artifactId>
+-      <version>1.0.0.0-SNAPSHOT</version>
++      <version>2.7.5.0.0</version>
+       <scope>provided</scope> <!-- for @ApiIgnore -->
+       <exclusions>
+         <exclusion>
+diff --git a/ambari-server/pom.xml.orig b/ambari-server/pom.xml.orig
```

Comment:
This patch seems unnecessary. Also, would you rename this file to a more explict/descriptive one? (and when renaming, note that Bigtop sorts patches in alphabetical order before applying)

Reply:
Yes, done.